### DISCUSSION TITLE
Print with one click

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15248] [How to] [answered]
Gents, I need to create workflow what will print some certain template right to assigned printer. What would be advice? PDF templates wouldn't work, is should be completely automated. To be used to print tickets. Appreciate guidance

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
> Gents, I need to create workflow what will print some certain template right to assigned printer. What would be advice? 

There is no ready functionality, you can try something like this: 
https://github.com/mike42/escpos-php
https://stackoverflow.com/questions/40998302/php-print-to-network-printer
>PDF templates wouldn't work,

What does not work ?
> is should be completely automated. To be used to print tickets. 

you are the first to ask, so "should" only in your opinion, no one else shows interest in it. 
no problem, we can do it just let someone order it
> Appreciate guidance

we appreciate, we ourselves have a head full of ideas, but we do not have time to implement them.

**Comment by s2b-git** [comment]
Sorry for unclear statement
> is should be completely automated. To be used to print tickets.
> you are the first to ask, so "should" only in your opinion, no one else shows interest in it.
> no problem, we can do it just let someone order it

Yes, I am describing my own certain scenario (Discussion marked as "How to"), I don't pass suggestion to set such functionality in the system core, I am asking for advice with hope out there is an easy solution on that  :-)

**Comment by JohnMaverick** [comment]
As Yetiforce  should be considered as a ERP/CRM/basic invoicing/accounting system, supporting POS printers and automated printing probably should be a part of core system. Connecting scanning barcodes would also welcome. This will extend scope of use-cases and give a light to wider group of corporate customers. Worth to think of it, but of course up to developer consideration
